apps/webstore/src/app/shared/components/modals/add-to-basket-modal/add-to-basket-modal.component.ts

```diff
@@ -280,6 +280,14 @@ export class AddToBasketModalComponent implements OnInit, OnChanges {
     }
   }
 
+  getDesignType(): string {
```

Comment:
This function is not necessary. You can call the desginType directly in the html and check what product type it is. Since there will be more product types coming in, there also need to be cases for new products.